spec/models/world_location_spec.rb

```diff
@@ -15,12 +17,22 @@
     end
   end
 
-  describe :== do
+  describe "#==" do
     let(:wl1) { WorldLocation.new({ "details" => { "slug" => "Neverland" } }) }
     let(:wl2) { WorldLocation.new({ "details" => { "slug" => "Neverland" } }) }
+    let(:wl3) { WorldLocation.new({ "details" => { "slug" => "Narnia" } }) }
+    let(:not_wl) { create(:organisation, slug: "Neverland") }
```

Comment:
```suggestion
    let(:not_wl) { build(:organisation, slug: "Neverland") }
```

Minor, switching to build is faster than create as it doesn't use the DB. Only a very minor optimisation in isolation but it's good practice to prefer build over create unless you actually need it.